### PR TITLE
35 implement removing profile pictures

### PR DESCRIPTION
**Pull request by Q-Basia**

### CODE REVIEW

File: code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/UploadImageActivity.java [hunk 1]
```diff
@@ -33,10 +38,12 @@
 public class UploadImageActivity extends AppCompatActivity {
 
     private ImageView imagePreview;
-    private Button upload_button, cancel_button, chooseImage_button;
-    private TextView prompt;
+    private Button upload_button, cancel_button, chooseImage_button, delete_button;
+    //    private TextView prompt;
```
Comment (dpardell) [inline code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/UploadImageActivity.java:42]: Make sure this isn't left commented out (either removed or implemented)

File: code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/UploadImageActivity.java [hunk 2]
```diff
@@ -113,5 +130,54 @@ public void onFailure(Exception exception) {
         });
 
         cancel_button.setOnClickListener(view -> this.finish());
+
+        delete_button.setOnClickListener(view -> {
+            Bitmap deterministicImage = ImageManager.generateDeterministicImage(userUuid);
+            ConfirmDeleteDialogFragment confirmDeleteDialogFragment = new ConfirmDeleteDialogFragment(imagePreview, deterministicImage, userUuid);
+            confirmDeleteDialogFragment.show(getSupportFragmentManager(), "confirmDelete");
+//            confirmDeleteDialogFragment.getDialog().setOnDismissListener(dialog -> {
```
Comment (dpardell) [inline code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/UploadImageActivity.java:138]: Same thing here about commented out code.

**Review by dpardell — COMMENTED** [review]
Overall looks good! I know this is still a draft, so feel free to disregard my comments if you were already about to deal with the stuff I was pointing out. Also, just a side note that I have revamped the ImageManagerTest file so if mine gets merged first, we will just need to deal with the conflict

**Review by Aryan-Rusia-2 — APPROVED** [review]
Looks good to me.

**Pull request merged**